ocs_ci/deployment/vmware.py

```diff
@@ -891,6 +891,21 @@ def generate_manifests(self):
             for each_file in files_to_remove:
                 os.remove(each_file)
 
+            # Update manifests for compact deployment if applicable
+            compact_deployment = config.ENV_DATA["worker_replicas"] == 0
+            if compact_deployment:
+                cluster_scheduler_config = os.path.join(
+                    self.cluster_path, "manifests", "cluster-scheduler-02-config.yml"
+                )
+
+                with open(cluster_scheduler_config, "r") as f:
+                    data = yaml.safe_load(f)
+
+                data["spec"]["mastersSchedulable"] = True
+
+                with open(cluster_scheduler_config, "w") as f:
+                    yaml.dump(data, f)
+
```

Comment:
AFAIK if we are setting worker count as 0 in install-config.yaml we don't need to make any changes to make master node schedulable